ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -433,20 +433,8 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
-- as: gcp-ipi-workload-identity-p2-f14
-  cron: 54 17 2,17 * *
-  remote_api: true
-  steps:
-    cluster_profile: gcp-qe
-    env:
-      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.11 and @gcp-ipi and not @fips'
-      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.11 and @gcp-ipi and not @fips'
-    test:
-    - ref: cucushift-installer-check
-    - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
-- as: gcp-ipi-disconnected-private-p2-f14
-  cron: 26 4 5,20 * *
+- as: gcp-ipi-disconnected-private-workload-identity-p2-f14
```

Comment:
In the source version - 4.11, there is no `gcp-ipi-disconnected-private-workload-identity-p2-f14` in openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml, are we going to add the same job in openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml?

Reply:
@jianlinliu sorry, just finished 4.14 - 4.12 (fresh install and upgrade), not yet touch 4.11 and 4.10, will do.